applications/incompressible_navier_stokes/vortex/application.h

```diff
@@ -366,7 +366,8 @@ class Application : public ApplicationBase<dim, Number>
     // momentum step
 
     // Newton solver
-    this->param.newton_solver_data_momentum = Newton::SolverData(100, 1.e-12, 1.e-6); // TODO
+    this->param.nonlinear_solver_data_momentum =
+      NonlinearSolver::SolverData(100, 1.e-12, 1.e-6, NonlinearSolver::SolverType::Newton); // TODO
```

Comment:
This is not my TODO ... I do not see, why not being able to converge to an extremely low tolerance for some cases is a problem?